deployments/local-minio/deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Network name to setup and deploy stack to. Also update instances in docker-compose.yml file is modified.
+networkname="cdr_local-minio"
+
+# Deploy docker-compose.yml file via docker stacks
+function deploy_images () {
+    printf "Deploying Docker Images...\n"
+    docker stack deploy -c docker-compose.yml $networkname
```

Comment:
✏️ Really small nit. The variable here is actually setting the name of the stack and not the name of the network.

Reply:
I was just naming the stack the same as the network to keep things simple, I updated the comment on the variable at the top a bit to give a better description.

Reply:
I'll rename it